test/fixtures/rules/std_rule_cases/L030.yml

```diff
@@ -54,3 +54,9 @@ test_pass_ignore_word:
     rules:
       L030:
         ignore_words: min
+
+test_pass_ignore_templated_code:
+  pass_str: |
+    SELECT
+        {{ "greatest(a, b)" }},
+        GREATEST(i, j)
```

Comment:
Could we add the same test to L010.yml?

It's possible we break L010, but override that breakage in L030.

Adding test for all 4 is probably overkill, but think we should add in L010 anyway (maybe instead of L030?)

Reply:
Ok, I added a similar test case to L010.yml. It's a bit different -- I tailored the test case to the rule.

```
{{ "select" }} a
FROM foo
WHERE 1
```